### DISCUSSION TITLE
Applying Icons

### DISCUSSION THREAD

**Discussion by sl-pk** [zixpo/candybar#180] [Q&A]
I've added the icons to ```drawable-nodpi``` and have added everything to the ```drawable.xml``` file. This shows all the icons in the preview section. But what do I have to do to actually apply them? I couldn't find the answer in the Wiki.

**Comment by Donnnno** [comment]
Did you add it to the appfilter.xml too?

**Reply by sl-pk** [reply]
I didn't, but I'm not sure what I have to add

**Reply by Donnnno** [reply]
You'll need to add the component info of the icons in order to apply them. You can gather it with icon request: https://play.google.com/store/apps/details?id=de.kaiserdragon.iconrequest

The drawable.xml is to make the apps visible inside the app.
